src/ripple/protocol/impl/STTx.cpp

```diff
@@ -161,7 +167,12 @@ STTx::getSigningHash () const
 uint256
```

Comment:
For a small performance improvement, consider changing this to `uint256 const&` and save yourself a 32-byte copy on every call.